workflow/rules/common.smk

```diff
@@ -179,5 +247,49 @@ def all_input(wildcards):
                 sample = sample
             )
         )
+        if not is_control(sample):
+            wanted_input.extend(
+                expand(
+                    [
+                        "results/deeptools/{sample}-{control}.plot_fingerprint.pdf",
+                        "results/macs2_callpeak/{sample}-{control}.{peak}_treat_pileup.bdg",
+                        "results/macs2_callpeak/{sample}-{control}.{peak}_control_lambda.bdg",
+                        "results/macs2_callpeak/{sample}-{control}.{peak}_peaks.{peak}Peak",
+                        "results/IGV/macs2_callpeak/{peak}/merged_library.{sample}-{control}.{peak}_peaks.igv.txt",
+                        "results/homer/annotate_peaks/{sample}-{control}.{peak}_peaks.annotatePeaks.txt",
+                        "results/macs2_callpeak/plots/plot_{peak}_peaks_count.pdf",
+                        # "results/intersect/plot_{peak}_peaks_frip_score.pdf",
+                        "results/macs2_callpeak/plots/plot_{peak}_peaks_macs2.pdf",
+                        "results/macs2_callpeak/plots/plot_{peak}_peaks_macs2_summary.txt",
+                        "results/homer/plots/plot_{peak}_annotatepeaks.pdf"#,
+                        # "results/homer/plots/plot_{peak}_annotatepeaks_summary.pdf"
+                    ],
+                sample = sample,
+                control = samples.loc[sample]["control"],
+                peak = config["params"]["peak-analysis"]
+            )
+        )
```

Comment:
I'd add one level of indentation, here, so that closing brackets line up with their opening statement:
```suggestion
                    sample = sample,
                    control = samples.loc[sample]["control"],
                    peak = config["params"]["peak-analysis"]
                )
            )
```